### DISCUSSION TITLE
v2.6.0

### DISCUSSION THREAD

**Discussion by MAKOMO** [artisan-roaster-scope/artisan#841] [Versions]
This version adds __performance and stability improvements as well as macOS and Windows legacy builds__

---
Release Sponsor: [Showroom Coffee]()  
<a target="_blank" href="https://showroomcoffee.com/">
<img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/showroom-logo.png?raw=true" width="500"></a>
---

The [Artisan project](https://artisan-scope.org/) runs on donations from individuals and companies recognising the value of Artisan for their work or leisure. Consider supporting this project with your [donation](https://www.paypal.com/paypalme/MarkoLuther), or even better, purchase an [artisan.plus membership](https://shop.artisan.plus/)!

## Documentation

__>>>  [Release Notes](https://artisan-roasterscope.blogspot.com/2022/03/artisan-v260.html) <<<__

See the [detailed release history](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/ReleaseHistory.md) for a complete list of additions, changes and fixes and the [Installation Instructions](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/Installation.md) for platform specific notes. Read our [Quick-Start Guide](https://artisan-scope.org/docs/quick-start-guide/)!

## Supported Platforms

Builds are available for macOS Catalina (10.15), Windows 10, Linux glibc 2.31 and 32bit Raspbian Bullseye or newer.

Legacy builds for macOS High Sierra (10.13), Windows 8 and Raspbian Bullet are available too.

---
PLEASE FOLLOW AND TAG US!  
&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://www.facebook.com/ArtisanScope"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/facebook-square.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.instagram.com/artisanscope/"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/instagram-square.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.youtube.com/channel/UCEp-DH_Y7sBR3-X3FWomgog"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/youtube-svgrepo-com.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://twitter.com/artisanscope"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/twitter-svgrepo-com.svg?raw=true" width="30"></a>&nbsp;&nbsp;&nbsp;<a href="https://www.linkedin.com/company/artisan-plus"><img src="https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/linkedin-svgrepo-com.svg?raw=true" width="30"></a>
---

__Licence Notes__

_Artisan is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version._

_Artisan is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE._

_Copies of the GNU General Public License has been included with this 
distribution in the file `LICENSE.txt`. An online version is available at
<http://www.gnu.org/licenses/>._

<hr /><em>This discussion was created from the release <a href='https://github.com/artisan-roaster-scope/artisan/releases/tag/'>v2.6.0</a>.</em>

**Comment by Jbletua** [comment]
Tengo un problema con Artisan 3.1.0 no me lee la temperatura

**Reply by poundy** [reply]
I am not sure why you posted on the release notice for V2.6.0 then? Please open a new issue. You will need to describe your problem, including what prior versions worked fine and what you did after the update to 3.1.0

**Reply by Jbletua** [reply]
Perfecto lo voy a publicar en artisan 3.1.0, pero te cuento antes usaba la versión 2.6.0 pero la computadora se arruino,compre otra Windows y instale la versión 3.1.0 pero cuando la conecto ami tostadora no me lee la temperatura
